### PR TITLE
Incorrect False positives and Risk accepted metrics

### LINKED ISSUE #4752: Incorrect False positives and Risk accepted metrics
**Slack us first!**
Hello. I write about problem here:
https://owasp.slack.com/archives/C2P5BA8MN/p1624892081234100

**Be informative**
As additional into slack I find the same behaviour with Risk Accepted findings. Into Metrics I see 0 **Risk Accepted** findings, but I have 1 Risk Accepted finding

**Bug description**
No error. Metrics into product, or metrics dushboard has incorrect info. 

**Steps to reproduce**
1. Create 2 two or more findings into product
2. SetUp **false Positive** for finding and **Risk Accepted** for other
1. Go to 'Metrics' of products see that here 0 'False Positives'
2. Go to Metrics - Simple Metrics
3. Click into _View Details_ of products
4. Scroll 
3. Scroll down to `Metric Counts` - `Accepted Findings` 
4. See here is 0

**Expected behavior**
I expected that I can see metrics with False Positives and RiskAccepted

**Deployment method** *(select with an `X`)*
- [X ] Docker
- [ ] Kubernetes
- [ ] GoDojo
- [ ] setup.bash / legacy-setup.bash

**Environment information**
 - Operating System: [e.g. Ubuntu 18.04]
 - DefectDojo is 1.15.0 from master

**Sample scan files** (optional)
If applicable, add sample scan files to help reproduce your problem.

**Screenshots** (optional)
DD_Metrics_Bug

**Console logs** (optional)
If applicable, add console logs to help explain your problem.

**Additional context** (optional)

### PR DESCRIPTION
**Pull request by XiChen-Tibco**
Fixes #4752 
Fix issue within simple matrix (Accepted Findings).
Display issue on "Inactive Findings" get fixed;
Also solved display issue on "Risk Accepted Matrix" within product page.

*Unable to reproduce the False positive issue within product matrix page.

### CODE REVIEW

**Review by Maffooch — APPROVED** [review]

File: dojo/product/views.py [hunk 1]
```diff
@@ -317,9 +317,8 @@ def finding_querys(request, prod):
     # risk_acceptances = Risk_Acceptance.objects.filter(engagement__in=Engagement.objects.filter(product=prod)).prefetch_related('accepted_findings')
     # filters['accepted'] = [finding for ra in risk_acceptances for finding in ra.accepted_findings.all()]
 
-    from dojo.finding.helper import ACCEPTED_FINDINGS_QUERY
```
Comment (valentijnscholten) [inline dojo/product/views.py:320]: a while ago we started to centralize the definition of queries for "accepted", "closed", etc so I'd prefer it to keep using these centralized Q objects.

**Comment by valentijnscholten** [comment]
For the record I think the original queries were made based on the requirement to show "findings accepted in this period". This PR changes changes this into "findings created in this period that are now accepted". Might make more sense.

File: dojo/product/views.py (hunk 1 above)
Comment (StefanFl) [reply dojo/product/views.py:320]: @valentijnscholten Do you request a change here or do we want to accept the PR?

File: dojo/product/views.py [hunk 2]
```diff
@@ -317,9 +317,8 @@ def finding_querys(request, prod):
...
-    filters['accepted'] = Finding.objects.filter(test__engagement__product=prod).filter(ACCEPTED_FINDINGS_QUERY).distinct()
-
+    filters['accepted'] = findings_qs.filter(date__range=[start_date, end_date],
+                                            risk_accepted=True)
```
Comment (valentijnscholten) [inline dojo/product/views.py:321]:
```suggestion
    from dojo.finding.helper import ACCEPTED_FINDINGS_QUERY
    filters['accepted'] = findings_qs.filter(date__range=[start_date, end_date], ACCEPTED_FINDINGS_QUERY)
```

**Review by valentijnscholten — CHANGES_REQUESTED** [review]

File: dojo/product/views.py (hunk 1 above)
Comment (valentijnscholten) [reply dojo/product/views.py:320]: just made the change, let's see if the tests go green

**Review by valentijnscholten — APPROVED** [review]

**Pull request merged**